### DISCUSSION TITLE
Spring Boot x Angular Entwicklung

### DISCUSSION THREAD

**Discussion by Marc738** [SE-TINF22B2/G3-ApeRepublic#104] [Technologien]
Discussion für hilfreiche Tutorials und Links zu Spring Boot und Angular...

**Comment by Marc738** [comment]
Beispielprojekt Spring Boot x Angular: [Inventory Management System](https://github.com/Akash-goyal-github/Inventory-Management-System.git)
Minimalaufwand Spring Boot x Angular: [WebMvcConfigurerAdapter](https://stackoverflow.com/questions/48503207/spring-boot-and-angular-5-project-structure)

**Comment by elnaz-gharoon** [comment]
Ich möchte am Samstag mehr über Angular lernen, es wäre gut, wenn ihr ein Tutorial hier vorschlagen.

**Reply by Marc738** [reply]
Kennst du da was @codajoda?
